### PR TITLE
[Edit] C++ Math Functions: `cosh()`

### LINKED ISSUE #979: [Edit] C++ Math Functions: `cosh()`
### Type of Edit (select all that apply)

Fix grammar, spelling, formatting
Clarify unclear content and/or terms
Add new content (definitions, codeblocks, etc.)

### Description (optional)

Requesting edits for the [cosh()](https://www.codecademy.com/resources/docs/cpp/math-functions/cosh) term entry under the C++: Math Functions concept.

Under `cpp/concepts/math-functions/terms/cosh/cosh.md`, please make the following edits:

- Revise the opening sentence to refer to the term "cosh()" by name (and **bold** this term, as well)
- In the "Syntax" section:
  - Rename any and all parameters to something more informative and descriptive, if possible.
  - If there is no text underneath the \```pseudo snippet, please add 1-2 sentences that describe the parameters of the `cosh()` function as well as the return value.
- Remove any phrasing that is instructive in tone, as opposed to informational. For example, something like "Use the `cosh()` function to..." should be revised to something akin to "The following example uses the `cosh()` function to..."
- If there is no Codebyte example, please add a new "Codebyte Example" section with 1-2 sentences previewing a \```codebyte/cpp code snippet.

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by kitrak-rev**
### Description

Closes #979 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by kitrak-rev** [comment]
@yangc95  Can you check and approve the workflow

**Comment by Dusch4593** [comment]
Hey @kitrak-rev make sure to check off those boxes under the "Checklist" at the top of this page to ensure you set up the PR correctly. 😄

File: content/cpp/concepts/math-functions/terms/cosh/cosh.md [hunk 1]
```diff
@@ -11,17 +11,25 @@ CatalogContent:
...
 
-Returns the hyperbolic cosine of an argument given in radians.
+The *cosh* function returns the hyperbolic cosine of an argument given in radians.
 
 ## Syntax
 
 ```cpp
 std::cosh(n)
 ```
+where `n` is the angle in `radians`
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/cosh/cosh.md:21]:
```suggestion
Where `n` is the angle in radians.
```

File: content/cpp/concepts/math-functions/terms/cosh/cosh.md [hunk 2]
```diff
@@ -11,17 +11,25 @@ CatalogContent:
...
+`Return type is double, float or long double.
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/cosh/cosh.md:22]:
```suggestion
Return type is double, float or long double.
```

File: content/cpp/concepts/math-functions/terms/cosh/cosh.md [hunk 3]
```diff
@@ -11,17 +11,25 @@ CatalogContent:
...
+
+If the magnitude of the result is too large in terms of the return type, the function returns HUGE_VAL with the proper sign, and an overflow range error occurs.
+
+## Mathematic Formula:
+cosh(x) =  (e <sup>x</sup> + e<sup>−x</sup> )/2
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/cosh/cosh.md:27]:
```suggestion
cosh(x) =  (e<sup>x</sup> + e<sup>−x</sup>)/2
```

File: content/cpp/concepts/math-functions/terms/cosh/cosh.md [hunk 4]
```diff
@@ -11,17 +11,25 @@ CatalogContent:
...
 
 ## Example 1
 
-Use `cosh()` to return the hyperbolic cosine of `0.0` radians:
+Use `cosh()` to return the hyperbolic cosine of `0.0` radians.
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/cosh/cosh.md:31]:
```suggestion
This example uses `cosh()` to return the hyperbolic cosine of `0.0` radians:
```

File: content/cpp/concepts/math-functions/terms/cosh/cosh.md [hunk 5]
```diff
@@ -37,3 +45,23 @@ int main() {
   // Output: The hyperbolic cosine of 0 radians is 1
 }
 ```
+
+
+## Example 2
+
+Use `cosh()` to return the hyperbolic cosine of `1.0` radians.
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/cosh/cosh.md:52]:
```suggestion
This example uses `cosh()` to return the hyperbolic cosine of `1.0` radians:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kitrak-rev I did a first review and left a few comments.

**Comment by SSwiniarski** [comment]
@kitrak-rev: This PR is a duplicate of [1069](https://github.com/Codecademy/docs/pull/1069), so I'm closing it.

**Pull request closed without merging**